libgc/include/gc.h

```diff
@@ -66,6 +61,10 @@
   typedef __int64 GC_signed_word;
 #endif
 
+#ifdef __cplusplus
```

Comment:
We have no interest in compiling libgc in C++. Revert all the changes in the `libgc/` folder.

Reply:
The libgc changes are needed for the sake of libgc callers.

Reply:
Mainline libgc already has all these changes, but presumably we are never updating.